core/src/test/java/nu/yona/server/test/util/JUnitUtil.java

```diff
@@ -121,4 +124,9 @@ public static Field getAccessibleField(Class<?> clazz, String fieldName)
 		field.setAccessible(true);
 		return field;
 	}
-}
+
+	public static void skipBefore(String description, ZonedDateTime now, int hour, int minute)
+	{
+		assumeThat(description, now, ZonedDateTimeMatchers.after(now.withHour(hour).withMinute(minute).withSecond(0)));
```

Comment:
I think it would be better to make the tests independent of "now" instead of skipping them at midnight. There could be interesting cases that fail at midnight that should not fail. I'll add a task on the board.

Reply:
That won't be trivial to do. To revive the tests under all circumstances and to also test such things around midnight, it would be better to mock the clock. The you have complete freedom about the scenario.